test/jenkins_tests/multi_node_docker_test.py

```diff
@@ -289,13 +305,15 @@ def run_test(self,
         Raises:
             Exception: An exception is raised if the timeout expires.
         """
+        print("Start Time: %s" % datetime.datetime.now())
```

Comment:
not a big deal, but if we specify what start time it is (e.g. `Multi-node docker test started at %s`), the message would be more understandable and easier for grep.